### DISCUSSION TITLE
github releases 未包含版本更新明细, 应至少添加链接到 CHANGELOG.md

### DISCUSSION THREAD

**Discussion by yoyo837** [hellof2e/quark-design#257] [General]
https://github.com/hellof2e/quark-design/releases 中添加固定链接到 https://github.com/hellof2e/quark-design/blob/main/packages/quarkd/CHANGELOG.md

**Comment by yoyo837** [comment]
不然每次都要翻文件夹打开几层.

**Comment by xxxsf** [comment]
收到！

**Comment by yoyo837** [comment]
尚未落地噢, 巨佬请考虑下支持.

**Reply by sanqi-med** [reply]
> 尚未落地噢, 巨佬请考虑下支持.

已落地，感谢建议

**Comment by xxxsf** [comment]
https://github.com/hellof2e/quark-design/releases/tag/v1.0.62

@yoyo837

**Reply by yoyo837** [reply]
巨佬, 请问发github releases是工具还是手动, 如果是手动, 我提要求就有点不好意思, 如果是工具, 给工具改一下就好了应该.

**Reply by yoyo837** [reply]
链接多了个右中括号`]`

**Reply by xxxsf** [reply]
你在刷新看

**Reply by yoyo837** [reply]
<img width="801" alt="image" src="https://github.com/hellof2e/quark-design/assets/6134547/3637a19c-6751-4096-aa4e-6e688bc35eda">

目前还在

**Reply by xxxsf** [reply]
嗯，已处理。感谢

**Reply by yoyo837** [reply]
Great, 谢谢!
